examples/ppo_sentiments.py

```diff
@@ -18,7 +18,7 @@ def get_positive_score(scores):
     return dict(map(lambda x: tuple(x.values()), scores))["POSITIVE"]
 
 
-default_config = yaml.safe_load(open("configs/ppo_config.yml"))
+default_config = yaml.safe_load(open("../configs/ppo_config.yml"))
```

Comment:
Revert this. We've been standardized on accessing paths from the root so we'd need to change this across all other examples. We can do this in a separate PR if you think it's worth it!